tests/config/pubsub/rabbitmq/pubsub.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: pubsub
+spec:
+  type: pubsub.rabbitmq
+  version: v1
+  metadata:
+  - name: host
+    value: "amqp://localhost:5672"
+  - name: consumerID
+    value: "testConsumer"
+  - name: durable
+    value: "false"
+  - name: deletedWhenUnused
+    value: "false"
+  - name: autoAck
+    value: "false"
+  - name: deliveryMode
+    value: "0"
+  - name: requeueInFailure
```

Comment:
should this be `requeueOnFailure`? should the metadata field name be changed?

Reply:
@artursouza that would be a breaking change ... 

or the other scenario ... add an alias and at a later point when we are making v2 of the component phase the old one out

Reply:
I don't think that changing this is enough to justify a new version of the component. If we want to fix this now, we should add a new metadata `requestOnFailure` and use that too. If user provides both, it should be a fatal error. Then, when we actually get to a v2 of this component, we can remove the old one.

How does that sound?

Reply:
Exactly my thoughts on breaking change. I think an alias can be provided and yes I agree that a fatal error can be logged if both are provided.

Reply:
OK. Let's do it as a separate PR. @pkedy Please, track this as a separate issue. I don't think is a priority for v1.1 to be honest.